Dockerfile-alpine

```diff
@@ -0,0 +1,108 @@
+FROM alpine:3.6
```

Comment:
alpine v3.12 is current stable

https://wiki.alpinelinux.org/wiki/Alpine_Linux:Releases

Reply:
Fixed